<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Table Of Contents</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Update the PDF file with this Table Of Contents</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSave">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="pdftoc.qrc">
              <normaloff>:/16x16/Save.png</normaloff>:/16x16/Save.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Pages</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="lblPages">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="txtBookmark"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Page</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="txtPage">
            <property name="inputMask">
             <string>9999</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Level</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="txtIndent">
            <property name="inputMask">
             <string>9999</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>PDF File</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Bookmark</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLineEdit" name="txtTitle"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnSaveBookmark">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="pdftoc.qrc">
              <normaloff>:/16x16/Save.png</normaloff>:/16x16/Save.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnInsertBookmark">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="pdftoc.qrc">
              <normaloff>:/16x16/Plus.png</normaloff>:/16x16/Plus.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnUpBookmark">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="pdftoc.qrc">
              <normaloff>:/16x16/Arrow2 - Up.png</normaloff>:/16x16/Arrow2 - Up.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDownBookmark">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="pdftoc.qrc">
              <normaloff>:/16x16/Arrow2 - Down.png</normaloff>:/16x16/Arrow2 - Down.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDeleteBookmark">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="pdftoc.qrc">
              <normaloff>:/16x16/Trash.png</normaloff>:/16x16/Trash.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="txtFileName"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoadFile">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="pdftoc.qrc">
          <normaloff>:/16x16/Folder.png</normaloff>:/16x16/Folder.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Quit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="pdftoc.qrc">
     <normaloff>:/16x16/Folder.png</normaloff>:/16x16/Folder.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="pdftoc.qrc">
     <normaloff>:/16x16/Save.png</normaloff>:/16x16/Save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="pdftoc.qrc">
     <normaloff>:/16x16/Close.png</normaloff>:/16x16/Close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pdftoc.qrc"/>
 </resources>
 <connections/>
</ui>
